src/nvim/tui/tui.c

```diff
@@ -374,6 +375,10 @@ static void tui_cursor_goto(UI *ui, int row, int col)
   unibi_goto(ui, row, col);
 }
 
+static void tui_update_menu(UI *ui)
+{
```

Comment:
Maybe a comment, like `// Do nothing; menus are for GUI-only`. (Assuming they are GUI-only. Are they?)